hack/load-tests/run-load-test.sh

```diff
@@ -360,15 +360,15 @@ function get_result_and_cleanup_fluentbit() {
 
 # shellcheck disable=SC2112
 function get_result_and_cleanup_selfmonitor() {
-   SCRAPESAMPLES=$(curl -fs --data-urlencode 'query=sum_over_time(scrape_samples_scraped{service="telemetry-self-monitor-metrics"}[5m]) / 300' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
+   SCRAPESAMPLES=$(curl -fs --data-urlencode 'query=sum_over_time(scrape_samples_scraped{service="telemetry-self-monitor-metrics"}[20m]) / 300' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
 
    SERIESCREATED=$(curl -fs --data-urlencode 'query=prometheus_tsdb_head_series{service="telemetry-self-monitor-metrics"}' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
```

Comment:
use max operator

Reply:
we need the ingestion rate here, the formula to calculate it is  either 
ingestion_rate = targets_count * metrics_per_target / scrape_interval or
ingestion_rate = active_time_series / scrape_interval